README.md

```diff
@@ -1 +1,30 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+This is the "Hello Microverse" Project with just Html and Css
```

Comment:
- [ ] As mentioned by the previous reviewer let's personalize this readme template to add "Hello Microverse" title instead of the placeholder "Project Name" here. Thank You :+1: